### PR TITLE
accounts_umask_etc_* and accounts_password_pam_minclass test scenarios

### PR DESCRIPTION
**Pull request by mildas**
#### Description:
New test scenarios for `accounts_umask_etc_profile`, `accounts_umask_etc_bashrc`, and `accounts_password_pam_minclass`.

Remove case insensitiveness from `accounts_umask_etc_bashrc` check - not aware of any system where e.g. `UMASK` works.

### CODE REVIEW

File: linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_etc_bashrc/tests/stig_correct.pass.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# profiles = xccdf_org.ssgproject.content_profile_stig
+
```
Comment (vojtapolasek) [inline linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_etc_bashrc/tests/stig_correct.pass.sh:3]: Please add here
# platform = Red Hat Enterprise Linux 8
cause the stig profile in rhel7 does not cotain this rule.

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
lgtm, thank you, see the comment.

File: linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_etc_bashrc/tests/stig_correct.pass.sh (hunk 1 above)
Comment (mildas) [reply linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_etc_bashrc/tests/stig_correct.pass.sh:4]: @vojtapolasek good point! Updated

**Review by vojtapolasek — APPROVED** [review]

**Pull request merged**